webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js

```diff
@@ -46,6 +38,14 @@ const ContentViewHistories = ({ cvId }) => {
 
 
   useDeepCompareEffect(() => {
+    const taskTypes = {
+      publish: 'Actions::Katello::ContentView::Publish',
+      promotion: 'Actions::Katello::ContentView::Promote',
+      removal: 'Actions::Katello::ContentView::Remove',
+      incrementalUpdate: 'Actions::Katello::ContentView::IncrementalUpdates',
+      export: 'Actions::Katello::ContentViewVersion::Export',
+    };
+
```

Comment:
Unrelated to the whole hooks rule but would it make more sense to have this in the Constants file?

Reply:
I can see that, but I feel like it makes sense to wait until it's used in multiple places.

Reply:
I feel the same way, but to be fair, I don't like that we have constants inside of our react components, I feel they should be placed above the component to prevent clutter. (I do understand the arguement of scope, but its the same memory footprint.